src/components/Navbar.tsx

```diff
@@ -1,101 +1,134 @@
+              height="2"
             />
-            <span className="text-center">{t("nav.zenikaSkills")}</span>
-          </div>
-        </Link>
-      </div>
-      <div className="w-1/3">
-        <Link href="/search">
-          <div className="flex flex-initial flex-col justify-between cursor-pointer">
-            {pathname === "/search" ? (
-              <Image
-                src={`/icons/${darkMode ? "dark" : "light"}/nav-selected.svg`}
-                alt={"Line"}
-                width="25"
-                height="2"
-              />
-            ) : (
-              <div className="h-px" />
-            )}
+          ) : (
+            <div className="h-px" />
+          )}
+          <Image
+            src={
+              pathname === "/zenika"
+                ? `/icons/${darkMode ? "dark" : "light"}/zenika-selected.svg`
+                : `/icons/${darkMode ? "dark" : "light"}/zenika.svg`
+            }
+            alt={"Line"}
+            width="25"
+            height="25"
+            className="p-1"
+          />
+          <span className="text-center">{t("nav.zenikaSkills")}</span>
+        </div>
+      </Link>
+      <Link href="/search">
+        <div className="flex flex-initial flex-col justify-between cursor-pointer">
+          {pathname === "/search" ? (
+            <Image
+              src={`/icons/${darkMode ? "dark" : "light"}/nav-selected.svg`}
+              alt={"Line"}
+              width="25"
+              height="2"
+            />
+          ) : (
+            <div className="h-px" />
+          )}
+          <Image
+            src={
+              pathname === "/search"
+                ? `/icons/${darkMode ? "dark" : "light"}/search-selected.svg`
+                : `/icons/${darkMode ? "dark" : "light"}/search.svg`
+            }
+            alt={"Line"}
+            width="25"
+            height="25"
+            className="p-1"
+          />
+          <span className="text-center">{t("nav.search")}</span>
+        </div>
+      </Link>
+      {user.email ===
+        config.nextPublicAdmins
+          .split(";")
+          .find((admin) => admin === user.email) && (
+        <Link href="/admin">
+          <div className="flex flex-initial flex-col justify-between cursor-pointer relative">
             <Image
               src={
-                pathname === "/search"
-                  ? `/icons/${darkMode ? "dark" : "light"}/search-selected.svg`
-                  : `/icons/${darkMode ? "dark" : "light"}/search.svg`
+                pathname === "/admin"
+                  ? `/icons/${darkMode ? "dark" : "light"}/zenika-selected.svg`
+                  : `/icons/${darkMode ? "dark" : "light"}/zenika.svg`
               }
-              alt={"Line"}
+              alt={"admin"}
               width="25"
               height="25"
               className="p-1"
             />
-            <span className="text-center">{t("nav.search")}</span>
+            {/* {skills && (
```

Comment:
Seems a bit heavy just to have the number of skills an admin have to add, do i still import skills to get the little "notification" ? 
If yes it's easy and fast to add :)

Reply:
Agree with you, a bit heavy for a little notification 😬 But it's requested, do you have another way in mind to do that ?

Reply:
The best would be a new request, like a select count, so you would just receive a number

Reply:
Sounds good, we can do this way